client/src/components/pages/home/PastTourneyCard.js

```diff
@@ -0,0 +1,52 @@
+import { Image, Typography } from "antd";
+import Text from "antd/lib/typography/Text";
+import React, { useState, useEffect } from "react";
+
+import ContentManager from "../../../ContentManager";
+
+import "./PastTourneyCard.css";
+import { Link } from "@reach/router";
+
+/**
+ * @typedef tourney
+ * @property {string} displayName
+ * @property {string} code
+ * @property {number} year
+ * @property {string} codeAndDivision
+ * @property {string} fullCode
+ */
+/**
+ * @param {object} props
+ * @param {tourney} props.tourney
+ */
+export default function PastTourneyCard({ tourney }) {
+  const [tourneyContent, SetTourneyContent] = useState(null);
+
+  useEffect(() => {
+    ContentManager.get(tourney.fullCode).then((content) => {
```

Comment:
i wonder what are the performance implications of individually loading every past tourney. we might want to consider only showing the first ~10 and having a "show more" button

Reply:
Would it be better to do some sort of scroll-based lazy loading? It might not be limited to past tournies either since it's already a somewhat long scroll even without that

Reply:
yeah it could be scroll-based or just a button, whatever you think is good and simple